docs/dev_guide/rtd.rst

```diff
@@ -0,0 +1,54 @@
+Using ReadTheDocs
+=================
+
+.. note::
+
+  For Tripal's own ReadTheDocs guidelines: :ref:`tripal_rtd`.
+
+
+What is ReadTheDocs?
+--------------------
+
+Documentation is important. It tells users how to use our product, and developers how to read our code.  We recommend hosting documentation for your custom module somewhere easily accessible, like ReadTheDocs.
+
+`ReadTheDocs <https://readthedocs.org/>`_ (RTD) uses the `Sphinx <http://www.sphinx-doc.org/en/master/>`_ framework to build a website in a Continuous Integration setup. Your RTD-compatible documentation is added directly to your module and when code changes are pushed to GitHub, the documentation website as defined by sphinx is built, and the "live" documentation website is updated.
+
+Benefits to housing documentation inside of your module code are:
+
+- Code changes can include documentation updates **in the same pull request**.
+- Changes to the documentation is **subject to review, just like code changes**.
+- Documentation changes are under **version control**.
+
+How do I add ReadTheDocs to my project?
+---------------------------------------
+Below is a quick overview of steps for integrating your module's documentation with RTD:
+
+- Set up your ReadTheDocs account and import your project.
+- Install Sphinx
+- Run the quickstart command: ``sphinx-quickstart``
+- Write your documentation (we're using reStructuredText (RST) format)
+  - Create an ``index.rst`` as the home page
+  ` Link other RST documents in your ``index.rst`` 
+- run ``make html`` in the docs folder to build your site for testing purposes. Sphinx will build you a searchable site with nicely formatted navigation.
```

Comment:
I think we're missing some steps here? How does the Makefile get in the docs directory?  Do we need this list of steps if it's not thorough?  If so, maybe leave out the 'make html' command?

Reply:
Ooops. added it twice.